modules/Makefile

```diff
@@ -88,7 +88,7 @@ MODULES_TO_DOCUMENT = \
 documentation: $(SYS_CTYPES_MODULE_DOC)
 	export CHPLDOC_AUTHOR='Cray Inc' && \
 	$(CHPLDOC) --save-sphinx ${MODULE_SPHINX} $(MODULES_TO_DOCUMENT)
-	cp -rf ${MODULE_SPHINX}/source/modules/standard ${DOC_SPHINX}/source/modules/
+	cp -rf ${MODULE_SPHINX}/source/modules/standard ${DOC_SPHINX}/source/modules/standard
```

Comment:
I must be missing something.  The old and new lines here should do the same thing.  (And in fact, in my repo without this patch, the cp -rf step here does put a standard subdir in .../source/modules/.)